src/wallet/wallet.cpp

```diff
@@ -4766,11 +4766,6 @@ void CWallet::MarkReserveKeysAsUsed(int64_t keypool_id)
     }
 }
 
-bool CWallet::HasUnusedKeys(int min_keys) const
```

Comment:
that's a part of https://github.com/bitcoin/bitcoin/pull/11044

Reply:
see https://github.com/dashpay/dash/pull/3096/commits/32a3adfa4f23a65dcd16cee8bb8ebf49b84debbb and https://github.com/dashpay/dash/pull/3096/commits/6946848e027927522e2bf1cc4d0d55d07e6b580b